.github/workflows/style.yml

```diff
@@ -32,25 +32,25 @@ jobs:
       - name: Python Code Quality and Lint
         # You may pin to the exact commit or the version.
         # uses: ricardochaves/python-lint@32032eca67291cd71f88d79e61bc4b904ee03265
-        uses: ricardochaves/python-lint@v1.3.0
+        uses: ricardochaves/python-lint@v1.4.0
         with:
           # A list of all paths to test
           python-root-list: "src/python/zquantum/core steps tests/zquantum/core tests/acceptance_tests"
 
           use-pylint: false
           use-pycodestyle: false
           use-flake8: true
-          # We're not ready yet to validate mypy, black & isort on every PR. In the 
+          # We're not ready yet to validate mypy on every PR. In the
           # mean time, please rely on local pre-commit hook checks.
           use-mypy: false
```

Comment:
There has been a point in time when z-quantum-core was also mypy-compatible. Would it make sense to at least try enabling this check, too?